pkg/devfile/devfile.go

```diff
@@ -341,20 +341,40 @@ func GetResourceFromDevfile(log logr.Logger, devfileData data.DevfileData, deplo
 
 					if currentPort > 0 {
 						servicePort := corev1.ServicePort{
+							Name:       strconv.Itoa(currentPort),
```

Comment:
I kinda remember the service port name cannot start with a number. so perhaps use `fmt.Sprintf(port-%v),currentPort)` instead?

Reply:
I found it https://kubernetes.io/docs/concepts/services-networking/service/

<img width="1760" alt="Screenshot 2023-08-15 at 3 25 10 PM" src="https://github.com/redhat-appstudio/application-service/assets/31771087/d9fc82cc-1067-45e6-84e4-e983950d1fc7">

Reply:
they can start and end with alphanumeric lowercase